spec/ping-api.json

```diff
@@ -0,0 +1,53 @@
+{
+  "openapi": "3.0.0",
+  "servers": [],
+  "info": {
+    "description": "This defines a simple API for checking if middleTier is alive.",
+    "version": "0.10.0-beta",
```

Comment:
@patrickmcelwee: we should change all these version to 1.0.0-rc.2

Reply:
yes, will do..